doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -111,15 +112,14 @@ Persistent Mounts
 To mount CephFS in your file systems table as a kernel driver, add the
 following to ``/etc/fstab``::
 
-    [{ipaddress}:{port}]:/ {mount}/{mountpoint} ceph [name=username,secret=secretkey|secretfile=/path/to/secretfile],[{mount.options}]
+    {name}@{fs_name}=/ {mount}/{mountpoint} ceph [mon_addr={ipaddress},secret=secretkey|secretfile=/path/to/secretfile],[{mount.options}]
 
 For example::
 
-    :/     /mnt/ceph    ceph    name=admin,noatime,_netdev    0       2
+    cephuser@cephfs=/     /mnt/ceph    ceph    mon_addr=192.168.0.1:6789,noatime,_netdev    0       2
```

Comment:
Specifying the `mon_addr` should not be typical for an fstab, right? Let's remove it.